BreweryRatings/Pages/Index.cshtml

```diff
@@ -5,6 +5,9 @@
     var businessesWithReviews = (List<Yelp.BusinessWithReview>)ViewData["BusinessesWithReviews"];
     var titleCity = ViewData["TitleCity"];
 }
+
+<img src='https://image.freepik.com/free-photo/close-up-blank-old-concrete-wall_23-2147856094.jpg' style='position:fixed;top:0px;left:0px;width:100%;height:100%;z-index:-1;'>
```

Comment:
A few notes here:
1) If you're linking to an image, you're stealing that company's bandwidth.  Better get permission to copy it locally.
2) Attribute values should be in double quotes, not single quotes.
3) For accessibility reasons, img tags should always have an alt property.